shared/actions/login/index.js

```diff
@@ -116,7 +118,7 @@ export function login (): AsyncAction {
                 payload: error
               })
 
-              if (!isRPCCancelError(error)) {
```

Comment:
This was not doing anything before becuase the stuff it was checking was inside `error.raw` I'm considering deleting this function from `engine/index` as well since now it's dead code